.travis.yml

```diff
@@ -19,6 +19,11 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 7.0
       env: COMPOSER_FLAGS="" SYMFONY_VERSION="3.0.*"
+    - php: 7.1
+      env: COMPOSER_FLAGS="" SYMFONY_VERSION="dev-master"
+  allow_failures:
+    - php: 7.1
```

Comment:
7.1 is tested with `--prefer-stable`

![image](https://cloud.githubusercontent.com/assets/1754678/25618787/9d604d22-2f48-11e7-9aa7-9d66c89a6b69.png)